exercises/phone-directory-cpp/src/singleton.cpp

```diff
@@ -1,18 +1,19 @@
-#include "../header/singleton.h"
+#include "../inc/singleton.h"
```

Comment:
If includepath is specified in CMake then it should be possible to add this include as:
#include "singleton.h"

Reply:
Fixed in last commit